query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/commons/LdapTestingUtils.java

```diff
@@ -202,4 +179,48 @@ private static BindRequest selectBindMethod(String bindMethod, String bindUser,
         reader.close();
         return entries;
     }
+
+    private static BindRequest selectBindMethod(String bindMethod, String bindUser, String password,
+            String realm, String kdcAddress) {
+        BindRequest request;
+
+        // TODO RAP 31 Jan 17: These case statements should operate in a case-insensitive manner
```

Comment:
If this switch case is using the enum constants, a case insensitive match isn't needed.